src/ore/src/metrics/delete_on_drop.rs

```diff
@@ -267,110 +247,8 @@ where
     }
 }
```

Comment:
Ideally we would have a single `impl<'a, V, L> Borrow<V::M> for DeleteOnDropMetric<'a, V, L>` but Rust doesn't let us do that because it fears conflicts with the blanket `impl<T> Borrow<T> for T`.